src/bot.jl

```diff
@@ -128,9 +128,15 @@ function BotConfig(
     precompiles_rootpath::AbstractString = "$(dirname(dirname(package_path)))/deps/SnoopCompile/precompile",
     subst::AbstractVector = Vector{Pair{UStrings, UStrings}}(),
     tmin::AbstractFloat = 0.0,
-    yml_path::Union{String, Nothing} = nothing
+    yml_path::Union{String, Nothing} = nothing,
+    blacklist = nothing       # deprecated keyword
     )
 
+    if blacklist !== nothing
+        Base.depwarn("`blacklist` is deprecated, please use `exclusions` to pass a list of excluded names")
```

Comment:
I think this will error 
```suggestion
        Base.depwarn(:blacklist, "`blacklist` is deprecated, please use `exclusions` to pass a list of excluded names")
```